src/tbp/plot/plots/interactive_hypothesis_space_correlation.py

```diff
@@ -49,15 +63,21 @@
 
 logger = logging.getLogger(__name__)
 
+vedo.settings.enable_default_keyboard_callbacks = False
 
 HUE_PALETTE = {
-    "Added": "#66c2a5",
-    "Removed": "#fc8d62",
-    "Maintained": "#8da0cb",
-    "Evidence": "#1f77b4",
-    "Slope": "#ff7f0e",
+    "Maintained": Palette.as_hex("numenta_blue"),
```

Comment:
Out of interest, did you try using the "Scientific Notation Colors" in the guide here? 

https://docs.google.com/document/d/1vcBChajQwYZnA76EkPbVd0LaBNiSqOsVOnqvkEsegiA/edit?usp=sharing

While maybe not as aesthetically pleasing, those are designed to be more easily distinguished, including by people with forms of color blindness. Unless you noticed a specific issue, I would therefore recommend we use those.